frontend/src/app/app.module.ts

```diff
@@ -9,8 +12,32 @@ import { AppRoutingModule } from './app-routing.module';
 
 @NgModule({
     declarations: [AppComponent],
-    imports: [BrowserModule, BrowserAnimationsModule, SharedModule, AppRoutingModule, MaterialModule],
-    providers: [],
+    imports: [
+        BrowserModule,
+        BrowserAnimationsModule,
+        SharedModule,
+        AppRoutingModule,
+        MaterialModule,
+        ReactiveFormsModule,
```

Comment:
Please check if it's the same as in registration

Reply:
we use `FormGroup` in both